### PR TITLE
302 goal setting dialog

### PR DESCRIPTION
**Pull request by bscheltinga**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/302

### CODE REVIEW

**Comment by bscheltinga** [comment]
Why are the Github actions not running?

**Comment by bscheltinga** [comment]
> Why are the Github actions not running?

Adding a new commit does not make the actions run...

**Comment by nelealbers** [comment]
@bscheltinga I think you need to resolve the conflicts first?

**Comment by bscheltinga** [comment]
@nelealbers yes you're right! Thanks

File: Rasa_Bot/domain/domain_goal_setting_dialogs.yml [hunk 1]
```diff
@@ -1,6 +1,19 @@
 version: "3.1"
 
+intents:
+  - goal_setting_dialog
+  - EXTERNAL_goal_setting_dialog
```
Comment (wbaccinelli) [inline Rasa_Bot/domain/domain_goal_setting_dialogs.yml:5]:
```suggestion
  - EXTERNAL_trigger_goal_setting

```
This is the definition in the nlu file. 
Also, we can get rid of the testing intent

File: Rasa_Bot/data/rules/rules_goal_setting_dialog.yml [hunk 2]
```diff
@@ -2,6 +2,65 @@
 version: "3.1"
 
 rules:
+
+
+## 1 Introduce goal setting dialog
+- rule: introduce goal setting dialog
+  steps:
+  - intent: goal_setting_dialog  # for testing
```
Comment (wbaccinelli) [inline Rasa_Bot/data/rules/rules_goal_setting_dialog.yml:10]: Probably we can use the external trigger already

File: Rasa_Bot/data/nlu.yml [hunk 3]
```diff
@@ -152,4 +152,15 @@ nlu:
...
+- intent: goal_setting_dialog
+  examples: |
+    - "goal setting dialog"
+    - "goal setting"
+    - "doel stellen"
+    - "gaol seting dialoq"
+    - "gool seeting diajlog"
+    - "gaal settting dialoog"
+    - "gol seting diaalog"
+    - "gal sating dialog"
```
Comment (wbaccinelli) [inline Rasa_Bot/data/nlu.yml:166]: We can use the external trigger and get rid of this, to make the training easier.

**Review by wbaccinelli — APPROVED** [review]
Great job!

File: Rasa_Bot/data/rules/rules_goal_setting_dialog.yml (hunk 2 above)
Comment (bscheltinga) [reply Rasa_Bot/data/rules/rules_goal_setting_dialog.yml]: I added the external trigger!

File: Rasa_Bot/data/nlu.yml (hunk 3 above)
Comment (bscheltinga) [reply Rasa_Bot/data/nlu.yml]: I removed the whole intent and only use the external trigger currently

**Pull request merged**